use std::io::Error;

fn main() -> Result<(), Error> {
    println!("Running queries ?");
    Ok(())
}
